features/checkout/addressing_order/choosing_province_for_country.feature

```diff
@@ -6,7 +6,7 @@ Feature: Choosing province for country
 
     Background:
         Given the store operates on a single channel in "United States"
-        And country "United States" has the "New York" province with "NY" code
+        And the country "United States" has the "New York" province with "NY" code
```

Comment:
@lchrusciel That is why our fixtures are probably wrong -> "The United States". But here the step is okay.
Since we are describing entities.